.github/workflows/e2e-arm64.yaml

```diff
@@ -0,0 +1,34 @@
+name: E2E-arm64
+on:
+  schedule:
+    - cron: '0 0/4 * * *' # runs every 4 hours
```

Comment:
4 hours is too frequent to me. I would suggest to run it daily.

Reply:
sure, change to daily, nightly test at 1 am.